response_operations_ui/templates/layouts/base.html

```diff
@@ -8,27 +8,49 @@
       <script type="text/javascript" src="{{ url_for('static', filename='js/main.min.js') }}"></script>
     {% endassets %}
     {% if config.GOOGLE_TAG_MANAGER %}
-      <!-- Google Tag Manager -->	
+      <!-- Google Tag Manager -->
       <script{% if csp_nonce %} nonce="{{ csp_nonce() }}"{% endif %}>
-        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':	
-        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],	
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=	
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl+'&gtm_auth={{ config.GOOGLE_TAG_MANAGER_PROP }}&gtm_cookies_win=x';f.parentNode.insertBefore(j,f);	
+        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        'https://www.googletagmanager.com/gtm.js?id='+i+dl+'&gtm_auth={{ config.GOOGLE_TAG_MANAGER_PROP }}&gtm_cookies_win=x';f.parentNode.insertBefore(j,f);
         })(window,document,'script','dataLayer','{{ config.GOOGLE_TAG_MANAGER }}');
-      </script>	
+      </script>
       <!-- End Google Tag Manager -->
     {% endif %}
 {% endblock %}
-{% block bodyStart %}	
+{% block bodyStart %}
   {% if config.GOOGLE_TAG_MANAGER %}
-    <!-- Google Tag Manager (noscript) -->	
-    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ config.GOOGLE_TAG_MANAGER }}&gtm_auth={{ config.GOOGLE_TAG_MANAGER_PROP }}&gtm_cookies_win=x"	
-    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>	
-    <!-- End Google Tag Manager (noscript) -->	
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ config.GOOGLE_TAG_MANAGER }}&gtm_auth={{ config.GOOGLE_TAG_MANAGER_PROP }}&gtm_cookies_win=x"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    <!-- End Google Tag Manager (noscript) -->
    {% endif %}
 {% endblock %}
-
-{% set pageConfig = { 
+{% if current_user.is_authenticated %}
```

Comment:
do we have test for this section? If not is it worth adding it?

Reply:
I added a test for this by tacking on a few extra checks in the login view tests.  Testing for 'my account' and 'sign out' when logged in, and the opposite when not authenticated